src/lib/molecules/AdvancedSearch/SearchResultRow.js

```diff
@@ -70,19 +70,20 @@ function SearchResultRow({ element }) {
                 })}
             >
                 {actions?.map((action, index) => (
-                    <Button
-                        key={index}
-                        tabIndex={0}
-                        size="medium"
-                        icon={action.icon}
-                        onBlur={handleBlur}
-                        appearance="minimal"
-                        onFocus={handleFocus}
-                        cornerRadius="smooth"
-                        onClick={() => action?.onClick(element)}
-                        className="searchResultRow__action"
-                        ariaLabel={action.name || action.icon}
-                    />
+                    <Tooltip key={index} text={action?.description} isVisible={!!action?.description}>
```

Comment:
You need to handle the `Tooltip` show logic also in case of user is focused on the `Button` using the tab key.
Need to change the `isVisible` prop to some state or memorized variable where you need to figure out the visibility state of the `Tooltip`.